charts/airbyte-server/templates/secrets.yaml

```diff
@@ -5,13 +5,10 @@ kind: Secret
 metadata:
   name: server-secrets
   labels:
-    app.kubernetes.io/name: {{ include "airbyte.name" . }}
-    app.kubernetes.io/instance: {{ .Release.Name }}
-    helm.sh/chart: {{ include "airbyte.chart" . }}
-    app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
-    app.kubernetes.io/managed-by: {{ .Release.Service }}
-    app.airbyte.io/fullname: {{ include "airbyte.fullname" . }}
+    {{- include "airbyte.labels" . | nindent 4 }}
   annotations:
+    helm.sh/hook: pre-install,pre-upgrade
```

Comment:
why do we need to add this?

Reply:
If you'll add secrets to your subchart, we need to make sure that secrets exists before deploying the app itself. Otherwise it'll error about secrets not found in the namespace

Reply:
got it. how come this wasn't erroring out before?